apps/fz_http/lib/fz_http_web/live/user_live/allowed_to_connect_component.ex

```diff
@@ -0,0 +1,28 @@
+defmodule FzHttpWeb.UserLive.AllowedToConnectComponent do
+  @moduledoc """
+  Handles user form.
+  """
+  use FzHttpWeb, :live_component
+
+  import Ecto.Changeset
+  alias FzHttp.Repo
+
+  @impl Phoenix.LiveComponent
+  def render(assigns) do
+    ~H"""
+    <input type="checkbox" phx-target={@myself} disabled={@user.role == :admin}
```

Comment:
Thoughts on adding `data-confirm="Are you sure? This may affect this user's internet connectivity."`